jenkins/job-configs/kubernetes-jenkins/jenkins-janitor.yaml

```diff
@@ -25,4 +25,14 @@
         - activate-gce-service-account
         - shell: |
             python $WORKSPACE/jenkins/janitor.py --project=sen-lu-test --hours=1
+            python $WORKSPACE/jenkins/janitor.py --project=kubernetes-pr-cri-validation --hours=6 --dryrun=true
```

Comment:
Let's have the default be safe? Aka add --dryrun=false for sen-lu-test and the others are true by default?